ui/src/app/components/apps-panel.tsx

```diff
@@ -78,10 +77,6 @@ export const AppsPanel = (props: {
     ...(config.enableCromwellGKEApp ? [UIAppType.CROMWELL] : []),
   ];
 
-  useEffect(() => {
```

Comment:
I think we still need this.  Consider a workspace which is open in multiple tabs.

Tab A: create app
Tab B - no information about the app being created, because the workspace hasn't been refreshed
Tab B, open the apps panel - still no update.  Here we need that info.